### PR TITLE
Update strings.po

### PR DESCRIPTION
**Pull request by prov3it**
# Check if this PR fulfills these requirements:
* [YES] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [NO] I have read the [**CONTRIBUTING**](../../master/Contributing.md) document.
* [NO] I made sure there wasn't another [Pull Request](../../../pulls) opened for the same update/change
* [NO] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [X] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description

The previous translation was really bad and taken way to literally from the English template, so i've made a lot of changes. I also made it more formal.

### CODE REVIEW

**Comment by CastagnaIT** [comment]
There is only a one rule for translations: https://github.com/CastagnaIT/plugin.video.netflix/issues/306
next time please respect it

If you have time look the translation @dagwieers

**Comment by CastagnaIT** [comment]
@prov3it please rebase you branch with last changes

File: resources/language/resource.language.nl_nl/strings.po [hunk 1]
```diff
@@ -150,41 +150,41 @@ msgstr "Handmatig ESN"
 
 msgctxt "#30035"
 msgid "InputStream Adaptive settings..."
-msgstr "InputStream Adaptive instellingen…"
+msgstr "InputStream aangepaste instellingen…"
```
Comment (michaelarnauts) [inline resources/language/resource.language.nl_nl/strings.po:153]: This shouldn't change.

File: resources/language/resource.language.nl_nl/strings.po [hunk 2]
```diff
@@ -352,39 +352,39 @@ msgstr "Bevat {} en meer..."
...
 
 msgctxt "#30095"
 msgid "All TV Shows"
-msgstr "Alle tvprogramma's"
+msgstr "Alle TV Programma's"
 
 msgctxt "#30096"
 msgid "All Movies"
-msgstr "Alle films"
+msgstr "Alle Films"
```
Comment (michaelarnauts) [inline resources/language/resource.language.nl_nl/strings.po:387]: Films should be lowercase

File: resources/language/resource.language.nl_nl/strings.po [hunk 3]
```diff
@@ -352,39 +352,39 @@ msgstr "Bevat {} en meer..."
...
 
 msgctxt "#30094"
 msgid "Browse recommendations and pick up on something new you might like."
-msgstr "Doorblader aangeprezen videos en kies iets dat je bevalt."
+msgstr "Blader door aangeprezen video's en kies iets dat je leuk lijkt."
 
 msgctxt "#30095"
 msgid "All TV Shows"
-msgstr "Alle tvprogramma's"
+msgstr "Alle TV Programma's"
```
Comment (michaelarnauts) [inline resources/language/resource.language.nl_nl/strings.po:383]: Should be tv-programma's

File: resources/language/resource.language.nl_nl/strings.po [hunk 4]
```diff
@@ -22,7 +22,7 @@ msgstr "Netflix"
 
 msgctxt "Addon Description"
 msgid "Netflix VOD Services Add-on"
-msgstr "Bekijk Netflix films en tvprogramma's in Kodi."
+msgstr "Bekijk Netflix films en TV Programma's in Kodi."
```
Comment (michaelarnauts) [inline resources/language/resource.language.nl_nl/strings.po:25]: Should be tv-programma's

File: resources/language/resource.language.nl_nl/strings.po [hunk 5]
```diff
@@ -960,67 +960,67 @@ msgstr "Top 10"
 
 msgctxt "#30243"
 msgid "Select first unwatched TV show episode"
-msgstr "Selecteer de eerste niet-bekeken tvprogramma aflevering"
+msgstr "Selecteer de eerste onbekeken TV Programma aflevering"
```
Comment (michaelarnauts) [inline resources/language/resource.language.nl_nl/strings.po:963]: tv-programma's

File: resources/language/resource.language.nl_nl/strings.po [hunk 6]
```diff
@@ -93,12 +93,12 @@ msgid "Rate on Netflix"
 msgstr "Op Netflix beoordelen"
 
 msgctxt "#30020"
-msgid "Remove from 'My list'"
```
Comment (michaelarnauts) [inline resources/language/resource.language.nl_nl/strings.po:96]: Has the msgid changed?

Comment (CastagnaIT) [reply resources/language/resource.language.nl_nl/strings.po:96]: no the id is not changed, original id must be keep

**Comment by michaelarnauts** [comment]
There are more occurrences of TV Programma's. These should all change to tv-programma's.

File: resources/language/resource.language.nl_nl/strings.po [hunk 7]
```diff
@@ -596,65 +596,65 @@ msgstr "Sorteer resultaten op"
...
 
 msgctxt "#30152"
 msgid "Suggestions for you"
-msgstr "Suggesties voor jou"
+msgstr "Aanbevolen voor jou"
 
 msgctxt "#30153"
 msgid "Year released"
-msgstr "Jaar van vrijgave"
+msgstr "Jaar vrijgegeven"
```
Comment (michaelarnauts) [inline resources/language/resource.language.nl_nl/strings.po:611]: I would use "Jaar van uitgave", I think this is a better wording.

**Comment by michaelarnauts** [comment]
The other changes are definitely an improvement. Thanks!

File: resources/language/resource.language.nl_nl/strings.po (hunk 7 above)
Comment (prov3it) [reply resources/language/resource.language.nl_nl/strings.po:611]: Im sorry im kinda new to this.

**Comment by prov3it** [comment]
> There is only a one rule for translations: #306
> next time please respect it
> 
> If you have time look the translation @dagwieers



> There is only a one rule for translations: #306
> next time please respect it
> 
> If you have time look the translation @dagwieers

Im sorry, im kinda new to this.

File: resources/language/resource.language.nl_nl/strings.po (hunk 4 above)
Comment (prov3it) [reply resources/language/resource.language.nl_nl/strings.po:25]: Agreed.

File: resources/language/resource.language.nl_nl/strings.po (hunk 1 above)
Comment (prov3it) [reply resources/language/resource.language.nl_nl/strings.po:153]: Okay.

File: resources/language/resource.language.nl_nl/strings.po (hunk 3 above)
Comment (prov3it) [reply resources/language/resource.language.nl_nl/strings.po:383]: Agreed.

File: resources/language/resource.language.nl_nl/strings.po (hunk 2 above)
Comment (prov3it) [reply resources/language/resource.language.nl_nl/strings.po:387]: Disagree. The words in msgid also start with a capital letter.

File: resources/language/resource.language.nl_nl/strings.po (hunk 7 above)
Comment (prov3it) [reply resources/language/resource.language.nl_nl/strings.po:611]: I searched for examples where movies are "uitgegeven". Books are, movies are not. So i dont agree.

**Review by prov3it — COMMENTED** [review]
> There are more occurrences of TV Programma's. These should all change to tv-programma's.
> 
> See: https://vrttaal.net/taaladvies-spelling/tv

I agree, however, the language iso codes are dutch, not flemish.

**Comment by CastagnaIT** [comment]
For the 30096 uppercase situation

@michaelarnauts do you think that english side should be changed as lowercase?
if so, for consistency i should change also all others menus as well, some hint appreciated

30174 "Tv Show genres" --> "TV show genres" ?
30187
30186
30095
30096

But it will be done not for this release (1.9.0) but next future release
translation changes are now locked

**Comment by michaelarnauts** [comment]
Hmm, it is common to capitalise words in English, like "All Movies" but this isn't done in Dutch "Alle films". I would keep the English version as it is.

File: resources/language/resource.language.nl_nl/strings.po (hunk 2 above)
Comment (michaelarnauts) [reply resources/language/resource.language.nl_nl/strings.po:387]: It's common to capitalise nouns in English, but it's also wrong. In Dutch, this isn't common and it is definitely wrong.

See https://www.grammarly.com/blog/capitalization-rules/

**Comment by michaelarnauts** [comment]
> > There are more occurrences of TV Programma's. These should all change to tv-programma's.
> > See: https://vrttaal.net/taaladvies-spelling/tv
> 
> I agree, however, the language iso codes are dutch, not flemish.

This isn't Flemish specific :) 

https://onzetaal.nl/taaladvies/afkorting-in-samenstelling/

File: resources/language/resource.language.nl_nl/strings.po (hunk 7 above)
Comment (michaelarnauts) [reply resources/language/resource.language.nl_nl/strings.po:611]: You might be right, but vrijgeven sounds odd for a movie, this is more used in context of something we get that has been released to the public. Maybe uitgebracht is better, but "Jaar van uitbrenging" sounds just silly :)

**Comment by CastagnaIT** [comment]
@prov3it let me know when the PR will be ready
and remember to rebase your branch with latest master branch changes

**Comment by CastagnaIT** [comment]
Tomorrow the new release will be rollout,
you have time until tomorrow, if it is not ready will be skipped for future releases

**Pull request closed without merging**